docs/dao/index.md

```diff
@@ -35,7 +35,7 @@ This section functions as the single source of truth of JAM Implementer DAO Memb
 | JAMdotTech \| PyJAMaz | 15gPiSBxhrrQFfShFbrnsQK7kokgQtVs3SEh38YjCDC31de3 | [Link](https://polkadot.subscan.io/extrinsic/25331656-2) |
 | MORUM | 123KEnwuHKiu48WsfwAX4YonGFTpkEmWS6V2WGoJA1qzBkbS | [Link](https://polkadot.subscan.io/extrinsic/25330323-2) |
 | Tessera | 15p3jWZaP4dHTkDTuKM5VXQL5XGfH9U5r6Ntu3UAv2K7vPb8 | [Link](https://polkadot.subscan.io/extrinsic/25331398-2) |
-| Fluffy Labs \| typeberry | 15fGrDWmFoaApnZXEw1Zg45zuCqpkbMS9YPzczeU9dHseUUu | [Link](https://polkadot.subscan.io/multisig_extrinsic/25345876-2) |
+| Fluffy Labs \| typeberry | 13Tomek8c9wxPJ7hFrbmXXHMzyJfcRWuxFjUbJLQgigRHAiv | [Link](https://polkadot.subscan.io/extrinsic/25373617-2) |
```

Comment:
Diffs shows up weird here; this change came from https://github.com/JamBrains/jam-docs/pull/22/commits/593e1f1a8e7d634eaf84eaf8244e500f2dc61b99